src/pcap/test/integration/agent_api_client.go

```diff
@@ -380,7 +380,7 @@ var _ = Describe("Using LocalResolver", func() {
 			targets = append(targets, agentTarget1)
 
 			agentTLSConf := &pcap.MutualTLS{SkipVerify: true}
-			apiBuffConf := pcap.BufferConf{Size: 100000, UpperLimit: 99000, LowerLimit: 90000}
```

Comment:
Why did you adjust the buffer configuration?

Reply:
I will drop that commit. The buffer size was fine / did not influence the result.